be/src/service/point_query_executor.cpp

```diff
@@ -80,6 +131,23 @@ Status Reusable::init(const TDescriptorTable& t_desc_tbl, const std::vector<TExp
         _col_uid_to_idx[slot->col_unique_id()] = i;
         _col_default_values[i] = slot->col_default_value();
     }
+
+    // Get the output slot descriptors
+    std::vector<SlotDescriptor*> output_slot_descs;
+    std::vector<vectorized::VExprSPtr> slot_exprs;
+    for (const auto& expr : _output_exprs_ctxs) {
+        extract_slot_ref(expr->root(), slot_exprs);
+    }
+    for (const auto& expr : slot_exprs) {
+        int column_id = static_cast<const vectorized::VSlotRef*>(expr.get())->column_id();
+        auto* slot_desc = tuple_desc()->slots()[column_id];
+        output_slot_descs.push_back(slot_desc);
```

Comment:
You can change `extract_slot_ref` to `extract_slot_desc` and extract `output_slot_descs` in it directlly.